docs/release-workflow.md

```diff
@@ -7,14 +7,34 @@ Reduce manually performed operation during csi release procedure.
 Use GitHub actions for release charts and update helm repository.
 
 Whole release workflow contains 2 parts:
-1. Release workflow in current repo triggered on release creation
+1. Release workflow in current repo triggered on release or pre-release creation
 2. Release workflow in [csi-baremetal-operator](https://github.com/dell/csi-baremetal-operator) repo
 
-### Detailed description
-
+## Detailed description
```

Comment:
Please remove Proposal word since this is documentation now. What steps should I do to release CSI charts?

Reply:
Done. I've clarified 'Usage' doc section, does it answer your question?